waspc/cli/src/Wasp/Cli/Command/Update.hs

```diff
@@ -0,0 +1,22 @@
+module Wasp.Cli.Command.Update
+  ( update,
+  )
+where
+
+import Control.Monad.IO.Class (liftIO)
+import System.Exit (ExitCode (..), exitWith)
+import System.Process (system)
+import Wasp.Cli.Command (Command)
+import Wasp.Cli.Command.Message (cliSendMessageC)
+import qualified Wasp.Cli.Common as Common
+import qualified Wasp.Message as Msg
+
+update :: Command ()
+update = do
+  cliSendMessageC $ Msg.Start "Updating wasp..."
+  status <- liftIO $ system "curl -sSL https://get.wasp-lang.dev/installer.sh | sh"
```

Comment:
Unfortunately, I don't think this will work, since wasp installer will refuse to write over existing files, and if we are doing updating, it will almost certainly write over existing files.

It will then let you know you need to run it with:
```
curl -sSL https://get.wasp-lang.dev/installer.sh | sh -s -- --force
```

So we can just go immediately with that, to make sure it actually performs the update?